zstacklib/zstacklib/storage/lvm/lv.py

```
@@ -0,0 +1,392 @@
+        Tuple of (return_code, stdout, stderr)
+        
+    Raises:
+        Exception: If target exists and overwrite is False
+    """
+    import time
+    
+    if not lv_exists(new_abs_path):
+        return bash.bash_roe(f"lvrename {old_abs_path} {new_abs_path}")
+
+    if not overwrite:
+        raise Exception(f"lv with name {new_abs_path} already exists, can not rename lv {old_abs_path} to it")
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)